OIN-SubOIN_in_PkioCert.md

```diff
@@ -7,7 +7,7 @@ Onderstaande beschrijving is voorgelegd aan de TSP's voor verificatie.
 </aside>
 
 ## Inleiding
-Dit hoofdstuk beschrijft Invulling (Sub)OIN en Organisatienaam, en –onderdeel/voorziening/samenwerking in velden van het PKIO certificaat [[Eisen Pkioverheid]] [[PKI PvE]] )
+Dit hoofdstuk beschrijft Invulling (Sub)OIN en Organisatienaam, en –onderdeel/voorziening/samenwerking in velden van het PKIO certificaat [[Eisen Pkioverheid]].
```

Comment:
voorstel is om verwijzing wel te laten staan naar onderdeel **3G** om aan te geven welk onderdeel precies relevant is

Reply:
"3g" staat nu in de tekst.